src/wazuh_db/wdb_global.c

```diff
@@ -599,6 +599,113 @@ int wdb_global_update_agent_group(wdb_t *wdb, int id, char *group) {
     }
 }
 
+int wdb_global_update_agent_groups_hash(wdb_t* wdb, int agent_id, char* groups_string) {
+    cJSON* j_agent_info = NULL;
+    char* agent_groups = NULL;
+    char truncated_groups_hash[9] = {0};
+    os_sha256 groups_hash;
+    int result = OS_INVALID;
+
+    // If the comma separated groups string is not send, read it from 'group' column
```

Comment:
Not sure whether we set this requirement, but I think that is good to have it in order to reuse the method when we want to fill the `groups_hash` column by taking into account the data in the `group` column.

Reply:
Great! This avoids an extra call to Wazuh-DB in `wdb_parse_global_update_agent_group()` to get the agent's group column. This method is writing that value and isn't necessary to read it.